app/Http/Controllers/Admin/RecommendedPlanController.php

```diff
@@ -0,0 +1,276 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Http\Controllers\Controller;
+use App\Models\Area;
+use App\Models\City;
+use App\Models\Prefecture;
+use Illuminate\Http\Request;
+use App\Models\Genre;
+use App\Models\Interest;
+use App\Models\Place;
+use App\Models\User;
+use App\Models\Keyword;
+use App\Models\PlaceImage;
+use App\Models\PlaceKeyword;
+use App\Models\Plan;
+use App\Models\PlanDetail;
+use App\Models\PlanFavorite;
+use App\Models\PlanKeyword;
+use Illuminate\Pagination\Factory;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Support\Facades\View;
+use Illuminate\Pagination\Paginator;
+
+class RecommendedPlanController extends Controller
+
+{
+    private $recommended_plan;
+
+    public function __construct(Plan $recommended_plan){
+        $this->recommended_plan = $recommended_plan;
+    }
+
+    //----- For RECOMMENDED PLANS LIST PAGE -----
+
+    # Use this to show the list of the Recommended plans
+    public function index(){
+        $recommended_plans = Plan::where('user_type', '=' , 'admin')->paginate(10);
```

Comment:
if the condition is just `equal(=)` there's no to specify `=` as second argument. this one will do `Plan::where('user_type', 'admin')->paginate(10);`